sources/AppBundle/Compta/BankAccount/BankAccountFactory.php

```diff
@@ -2,38 +2,33 @@
 
 namespace AppBundle\Compta\BankAccount;
 
-use Afup\Site\Utils\Configuration;
-
 class BankAccountFactory
 {
-    /**
-     * @var Configuration
-     */
-    private $configuration;
-
-    public function __construct(Configuration $configuration)
-    {
-        $this->configuration = $configuration;
-    }
-
     public function createApplyableAt(\DateTimeInterface $applicationDate = null)
     {
-        if (null === $applicationDate) {
-            $configKey = 'rib';
-        } else {
-            $comparisonDate = \DateTime::createFromFormat('Y-m-d', $this->configuration->obtenir('rib_ce|valid_until'));
-            $comparisonDate->setTime(0, 0, 1);
-            $configKey = $applicationDate <= $comparisonDate ? 'rib_ce' : 'rib';
+        $comparisonDate = new \Datetime('2021-01-01');
+        $comparisonDate->setTime(0, 0, 1);
+
+        if (null === $applicationDate || $applicationDate > $comparisonDate) {
+            return new BankAccount(
+                '06076',
+                '00020707701',
+                '00020707701',
+                '70',
+                "CCM PARIS MAGENTA GARE DE L'EST",
+                'CMCIFR2A',
+                'FR76 1027 8060 7600 0207 0770 170'
+            );
```

Comment:
```suggestion
            return new BankAccount(
                '10278',
                '06076',
                '00020707701',
                '70',
                "CCM PARIS MAGENTA GARE DE L'EST",
                'CMCIFR2A',
                'FR76 1027 8060 7600 0207 0770 170'
            );
```

En prod on a ces valeurs : 
```
$configuration['rib']['etablissement']='10278';
$configuration['rib']['guichet']='06076';
$configuration['rib']['compte']='00020707701';
$configuration['rib']['cle']='70';
$configuration['rib']['domiciliation']="CCM PARIS MAGENTA GARE DE L'EST";
$configuration['rib']['bic']='CMCIFR2A';
$configuration['rib']['iban']='FR76 1027 8060 7600 0207 0770 170';

```